.github/workflows/yaml-lint.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: 'Yamllint'
         uses: karancode/yamllint-github-action@master
         with:
-          yamllint_file_or_dir: 'charts/whatsapp-proxy-chart/*/*'
+          yamllint_file_or_dir: 'charts/whatsapp-proxy-chart/*/*.yml'
```

Comment:
Yeah we need to add `*.yaml` here as well because almost all of our files are `.yaml`s.

Reply:
Thinking about it more, I'm pretty sure helm charts actually *fail* yamllint, since the interpolation isn't yaml standard. I don't think we can use this, we'll need to do a `helm lint` instead.

Reply:
We should use https://github.com/helm/chart-testing-action instead. Although neither of these would catch the `procotcl` misspelling error, as there's no defined spec for it. It'd catch invalid *formatting*, but not invalid keys.